tee/tf/main.tf

```diff
@@ -0,0 +1,277 @@
+terraform {
+  backend "s3" {
+    bucket = "proving-server-tf-state"
+    key    = "terraform/tee/terraform.tfstate"
+    region = "eu-north-1"
+  }
+}
+
+variable "aws_region" {
+  description = "AWS region to deploy to"
+  type        = string
+  default     = "eu-north-1"
+}
+
+variable "instance_type" {
+  description = "EC2 instance type for Nitro Enclave"
+  type        = string
+  default     = "c5.xlarge"
+}
+
+variable "key_name" {
+  description = "(Optional) EC2 key pair name for SSH access"
+  type        = string
+  default     = ""
+}
+
+variable "eif_url" {
+  description = "Public URL of the Enclave Image File (EIF)"
+  type        = string
+}
+
+variable "proxy_image" {
```

Comment:
What about changing names of variables from generic `proxy*` to `prover-server-proxy*`? As when I see in the EC22 console, I see generic name `proxy-instance`, but this is really a prover server
<img width="214" height="171" alt="image" src="https://github.com/user-attachments/assets/c936d8f9-ebf4-417b-8cd9-c1c5789c1e72" />

Reply:
After I thought, please ignore this one